GATK/4.1.3.0/CollectMultipleMetrics.nf

```diff
@@ -0,0 +1,32 @@
+
+process CollectMultipleMetrics {
+  tag {"CollectMultipleMetrics ${sample_id}"}
+  label 'GATK'
+  clusterOptions = workflow.profile == "sge" ? "-l h_vmem=${params.collectmultiplemetrics_mem}" : ""
+  publishDir "$params.out_dir/$sample_id/multiple_metrics", mode: 'copy'
+
+
+  input:
+    tuple sample_id, file(bam)
+
+  output:
+    file ("multiple_metrics*")
+
+  script:
+  """
+  gatk --java-options "-Xmx${task.memory.toGiga()-4}g -Djava.io.tmpdir=\$TMPDIR" \
+  CollectMultipleMetrics \
+  -I $bam \
+  -O multiple_metrics\
+  -R $params.genome_fasta \
+  --PROGRAM CollectAlignmentSummaryMetrics \
```

Comment:
Maybe make 'programs' a parameter (list of programs)?

Reply:
As discussed earlier, I think we should assign this to a toolOptions variable in the config, like so:
```suggestion
 toolOptions = "
  --PROGRAM CollectAlignmentSummaryMetrics \
  --PROGRAM etc \
 "
 
```